src/pj_factors.c

```diff
@@ -72,7 +72,7 @@ int pj_factors(LP lp, PJ *P, double h, struct FACTORS *fac) {
 
         /* convergence */
         if (!(fac->code & IS_ANAL_CONV)) {
-            fac->conv = - atan2(fac->der.y_l, fac->der.x_l);
+            fac->conv = - atan2(fac->der.x_p, fac->der.y_p);
```

Comment:
I'm not sure why I'm looking at that as I've absolutely no background on this :-) , but this change looks surprising since atan2 arguments are documented to be in y,x order.